### ISSUE TITLE
Persian pretrained models

### ISSUE THREAD

**Issue by Dmitry-Sn** [Helsinki-NLP/Opus-MT#33]
I want to find any pretrained models paired with Persian (Farsi). In the matrix view of the downloadable models, I see that you have fa-fi and fi-fa pairs, but the links to their models do not work. How can I download them?

**Comment by jorgtied** [comment]
Does it work now? I may also add some more models combined with Farsi. Any particular language pair you are interested in?

**Comment by Dmitry-Sn** [comment]
Yes, it works great, thanks a lot.
Of those that are presented in the matrix of models, I tried all the most interesting ones (except for Persian pairs so far). Outside of the matrix, I'm interested in pairs (Turkish / Persian) -> (English / Russian). I liked the Turkish-English model and I have already tried to build Turkish-Russian through various intermediate languages, and I was amazed with the result here too. But maybe you have models that are not currently represented in the matrix of downloadable models?
